packages/graphback-codegen-client/src/templates/gqlTemplates.ts

```diff
@@ -28,23 +28,23 @@ export const findOneQuery = (t: ModelDefinition) => {
     ${fieldName}(id: $id) {
       ...${t.graphqlType.name}ExpandedFields
     }
-  }`
+         }`
```

Comment:
This is in line with the `q` in this function, but that does not necessarily mean they will line up the same way in the output file.

Read this article to help you understand how to fix the problem:

https://2ality.com/2016/05/template-literal-whitespace.html

As always, if you are still stuck we are here to help!

Reply:
I am still not able to solve the issue after yarn test 17 test are failing even if I have undone all my changes. Solution in this article https://2ality.com/2016/05/template-literal-whitespace.html is to return query like this image.
![Screenshot 2020-09-24 at 5 32 40 PM](https://user-images.githubusercontent.com/50217679/94142505-04f0fc00-fe8c-11ea-87bc-3df9a5caa143.png)

Reply:
You don't need to use any special helpers or anything other than what is already there - you are on the right track in that you just need to position the closing bracket `}` in the right place. You are putting it on the wrong place - if you can figure out where it will go then it should work.

Reply:
I am getting 17 errors while doing yarn test, how can I check if output is correct or not, after doing the changes?

Reply:
From the `templates/ts-react-apollo-client` directory, run `yarn graphback generate`.

Note, that you need to change the path of `extensions.graphback.model` in the `.graphqlrc` file to point to a valid model file ([this one](https://github.com/aerogear/graphback/blob/f262f4cd5d36dc1ef13e9d7188ce9ff7389898ab/templates/ts-apollo-postgres-backend/model/datamodel.graphql#L21)).

Reply:
Gneration Successful! what should I do next? Sorry I have no past experience of open-source so I will need some help in the starting.

Reply:
Cool! So the generated file is `graphback.graphql` - can you check if the `}` are indented or at the start of line (check the issue for how it should look).

No problem, we will get through this!

Reply:
I have updated the original issue description to make it clearer (see the code block).